sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/CustomEvents/CustomEventLogger.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Monitor.OpenTelemetry.Events
```

Comment:
Just curious, how did we land on this namespace?

It's unusual because it doesn't match the project & directory (Azure.Monitor.OpenTelemetry.Exporter.CustomEvents).
Second, all our other public classes are in the root directory (Azure.Monitor.OpenTelemetry.Exporter).

Reply:
It is going to be used in both distro and exporter so wanted something common. I am not sure if namespace matching the directory/project is a requirement.  Found at least one similar example: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs.Batch/api/Azure.Storage.Blobs.Batch.netstandard2.0.cs